.github/workflows/kotlin-test.yml

```diff
@@ -0,0 +1,64 @@
+name: Kotlin CI
+
+on:
+  push:
+    branches-ignore:
+      - master
+    paths: [ "app-android/**" ]
+
+jobs:
+  test-kotlin:
+    runs-on: ubuntu-latest
+    env:
+      java-version: 21
+      java-distribution: 'temurin'
+      node-version: 20.11.1
+      TZ: "Europe/Berlin"
+    permissions:
+      actions: none
+      checks: none
+      contents: read
+      deployments: none
+      id-token: none
+      issues: none
+      discussions: none
+      packages: none
+      pages: none
+      pull-requests: none
+      repository-projects: none
+      security-events: none
+      statuses: none
+
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 #v4.1.1
+      - name: Use Node.js ${{ env.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ env.node-version }}
+          cache: 'npm'
+          cache-dependency-path: '**/package-lock.json'
+      - name: Get better-sqlite3 cached location
+        run: |
+          echo "better_sqlite3_path=$(node buildSrc/getNativeCacheLocation.js better-sqlite3)" >> $GITHUB_ENV
+      - name: Try to use cached better-sqlite3
+        uses: actions/cache@v3
+        with:
+          path: ${{ env.better_sqlite3_path }}
+          key: ${{ env.better_sqlite3_path }}
+      - name: Install packages
+        run: npm ci && npm run build-packages
+      - name: Build WebApp
```

Comment:
do you really need to build the webapp to run tests? I don't think so tbh

Reply:
Unfortunately yes, it tries to find the webapp inside the assets and if it doesn't, it creates a file pointing to app's build folder and take the files from there

Reply:
I deleted `build` folder for the webclient and then successfully run `./gradlew test`. Am I missing something?